app/views/users/show.html.erb

```diff
@@ -1 +1,39 @@
-<h1>These are a user for a Id<h1>
+<h1>These are a user for Information</h1>
+
+    <div class= 'styleusers'>
+        <%= image_tag @user.photo,alt: "image user" %>
+        <div class='title'>
+               <h2><%= @user.name%></h2>
+                <div class = 'counter'>
+                    <p>posts counters:<%= @user.postscounter%></p>
+                </div>
+        </div>
+    </div>
+    <div class="bioinfo">
+        <h3>Bio</h3>
+        <p><%= @user.bio %></p>
+    </div>
+    <div class= 'wraperusers'>
+        <% @user.recents_post.each do |post| %>
+            <div class='titleposts'>
+               <h4>Post:<%= post.id%> <%=post.title%></h4>
+               <p><%= post.text%></p>
+               <p>
+                <div class = 'counter'>
+                    <p>comments:<%=post.commentscounter%></p>
+                    <p>likes: <%=post.likescounter%></p>
+                </div>
+        </div>
+        <% end %>
+        <button class='buttonstyle' type='button'>
+        <%=link_to user_posts_path(@user) do%>
+        See all Posts
```

Comment:
- [ ] I have created two users **Tom** and **Lily**, I created one post for tom and on that post I have created a comment from Lily. However when I go to Lily's  show page it shows the Tom's post which it should not be because it is not Lily's post. Also when I click check `see all posts` button on Lily's show page it throws me error because it can't find posts since there are none. Kindly fix this issue
![image](https://user-images.githubusercontent.com/62705947/214685005-5fc79677-b6c3-4153-a45e-90713d158eb9.png)

Reply:
hi @GzimAsani  i deploy the lily user and show me the information about it

Reply:
Let me record a video

Reply:
![image](https://user-images.githubusercontent.com/62705947/214691916-09502420-da50-4488-b9c1-302c743eb2cd.png)
![image](https://user-images.githubusercontent.com/62705947/214692468-19d8a2f0-2b85-4ee0-95e6-ac67eb8f73a0.png)
https://www.loom.com/share/c4d7074e5fb14b03b7c767ddc17a7fb2

Reply:
thank you i see the error